src/main/java/org/killbill/billing/plugin/core/config/PluginEnvironmentConfig.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2014-2017 Groupon, Inc
+ * Copyright 2014-2017 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.core.config;
+
+import java.util.Properties;
+
+public class PluginEnvironmentConfig {
+
+    public static final String KILL_BILL_NAMESPACE = "org.killbill.";
+
+    public static String getRegion(final Properties properties) {
+        // See KillbillServerConfig
+        return properties.getProperty(KILL_BILL_NAMESPACE + "server.region");
+    }
+}
```

Comment:
It is actually setup that way (the `properties` would come from `OSGIConfigProperties`).

While in the core we rely on config-magic to not rely on property keys (we use methods on the config POJOs), once we go into OSGI world, we simply pass the raw `Properties` map as `OSGIConfigProperties`. Unless I'm missing something, we pretty much have to go back to knowing about these keys.